README.md

```diff
@@ -6,29 +6,40 @@
 
 # Movie Site Project
 
-Replace this readme with your own information about your project.
+Build a movie site 🍿
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is my first multi-page React application and use a fun new API.
 
-## Getting Started with the Project
+The goal is to list popular movies by using themoviedb.org's API to fetch a list of movies and show details about them on a seperate page by using React and React Router.
 
-### Dependency Installation & Startup Development Server
+## Highlights:
 
-Once cloned, navigate to the project's root directory and this project uses npm (Node Package Manager) to manage its dependencies.
+I try to fullfill all the requirements and add a PageNotFound page when users try to visit a movie detail page with an invalid movie ID.
 
-The command below is a combination of installing dependencies, opening up the project on VS Code and it will run a development server on your terminal.
+On the homepage where I list Now Playing movies and on mobile screen, I add a dropdown to change the list between popular, top rated and upcomning.
 
-```bash
-npm i && code . && npm run dev
-```
+### Future Improvements
 
-### The Problem
+Here are several enhancements I’d like to implement with more time: 
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+  - Favoratie movie list:
+    I would like to add an fav-movie page in the MovieList and complete the stretch goals.
 
+  - Light & Dark mode
+
+  - Search input:
+    that user could search for movies from the list
+  
+  - Play button
+
+  - Handle loading states: streth goals
+
+  - Work on with the style and try to use Styled components in React
```

Comment:
Yes, I agree, if you could start using styled components or Tailwind it would benefit you with more flexibility, efficiency, and consistency for React development and not as hard to maintain as seperate CSS files is, especially in larger projects and if you want to keep on developing and expanding this page.